### PR TITLE
US-631955 changes to support deploying pods without cpu limit

### PR DESCRIPTION
**Pull request by NekhilKotha**

### CODE REVIEW

**Comment by kishorv10** [comment]
@NekhilKotha The same requirement is addressed in PR #759 . With this change, values.yaml configuration can be used to set or not set CPU limits for Pega work loads. Please try, if that solves your usecase.

File: charts/pega/templates/_pega-deployment.tpl [hunk 1]
```diff
@@ -225,9 +225,9 @@ spec:
           # Maximum CPU and Memory that the containers for {{ .name }} can use
           # Resources are configured through deprecated settings. Use .tier[].resources instead
           limits:
-          {{- if .node.cpuLimit }}
+          {{- if and .node.cpuLimit (ne (include "deployWithoutCPULimit" (dict "node" .node)) "true") }}
```
Comment (Saurabh-16) [inline charts/pega/templates/_pega-deployment.tpl:228]: Can we simplify this in below sequence:-
If .node.cpuLimit is given, we always honour this.
else if we check for deploywithoutCPULimit
else we default it to 4 ?

Comment (Saurabh-16) [reply charts/pega/templates/_pega-deployment.tpl:228]: Also how  CMC customers will be aware of this feature? Is there plan to provide some documentation around it.

**Comment by NekhilKotha** [comment]
> @NekhilKotha The same requirement is addressed in PR #759 . With this change, values.yaml configuration can be used to set or not set CPU limits for Pega work loads. Please try, if that solves your usecase.

Hi @kishorv10, I have initially thought of using the same but we need this control of CPU Limit conditionally for controlplane Integration. So this is not possible to use that

**Comment by kishorv10** [comment]
@NekhilKotha I believe with the latest developments, this PR may not be needed. Can you please confirm & close it accordingly?

**Pull request closed without merging**